src/gui/plugins/visualization_capabilities/VisualizationCapabilities.cc

```diff
@@ -0,0 +1,2544 @@
+    /// \param[in] _inertial Inertial component of the link
+    /// \param[in] _parent Visual parent
+    /// \return Visual (center of mass) object created from the inertial
+    public: ignition::rendering::VisualPtr createCOMVisual(
+      ignition::gazebo::Entity _id,
+      const math::Inertiald &_inertia,
+      ignition::rendering::VisualPtr &_parent);
+    /////////////////////////////////////////////////
+    // Inertia
+    /////////////////////////////////////////////////
+    /// \brief View inertia of specified entity
+    /// \param[in] _entity Entity to view inertia
+    public: void ViewInertia(const Entity &_entity);
+
+    /// \brief Callback for view inertia request
+    /// \param[in] _msg Request message to set the target to view inertia
+    /// \param[in] _res Response data
+    /// \return True if the request is received
+    public: bool OnViewInertia(const msgs::StringMsg &_msg,
+      msgs::Boolean &_res);
+
+    /// \brief Create an inertia visual
+    /// \param[in] _id Unique visual id
+    /// \param[in] _inertial Inertial component of the link
+    /// \param[in] _parent Visual parent
+    /// \return Visual (center of mass) object created from the inertial
+    public: ignition::rendering::VisualPtr CreateInertiaVisual(
+      ignition::gazebo::Entity _id,
+      const math::Inertiald &_inertia,
+      ignition::rendering::VisualPtr &_parent);
+
+    /////////////////////////////////////////////////
+    // Joints
+    /////////////////////////////////////////////////
+
+    /// \brief Callback for view joints request
+    /// \param[in] _msg Request message to set the target to view center of
+    /// mass
+    /// \param[in] _res Response data
+    /// \return True if the request is received
+    public: bool OnViewJoints(const msgs::StringMsg &_msg,
+      msgs::Boolean &_res);
+
+    /// \brief View joints of specified entity
+    /// \param[in] _entity Entity to view joints
+    public: void ViewJoints(const Entity &_entity);
+
+    /// \brief Create a joint visual
+    /// \param[in] _id Unique visual id
+    /// \param[in] _joint Joint sdf dom
+    /// \param[in] _childId Joint child id
+    /// \param[in] _parentId Joint parent id
+    /// \return Visual (joint) object created from the sdf dom
+    public: rendering::VisualPtr CreateJointVisual(Entity _id,
+        const sdf::Joint &_joint, Entity _childId = 0,
+        Entity _parentId = 0);
+
+    /// \brief Updates the world pose of joint parent visual
+    /// according to its child.
+    /// \param[in] _jointId Joint visual id.
+    public: void UpdateJointParentPose(Entity _jointId);
+
+    /////////////////////////////////////////////////
+    /// \brief Ignition communication node.
+    public: transport::Node node;
+
+    /// \brief Keep track of world ID, which is equivalent to the scene's
+    /// root visual.
+    /// Defaults to zero, which is considered invalid by Ignition Gazebo.
+    public: Entity worldId{0};
```

Comment:
We should use `kNullEntity`, because the exact value is not guaranteed to remain the same.

```suggestion
    public: Entity worldId{kNullEntity};
```

Reply:
0a8b9e7